package.json

```diff
@@ -5,37 +5,38 @@
   "scripts": {
     "lint": "eslint --ignore-path .gitignore --ext js,ts .",
     "build": "lerna run build",
-    "test": "lerna run test --parallel",
+    "test": "vitest run && lerna run test --parallel",
```

Comment:
Run vitest at a parent level as this is what it will become in the future. We still run the lerna command as there is a post test script in the core package. In the next step this script will move up.